homeassistant/components/remote_calendar/config_flow.py

```diff
@@ -50,6 +51,13 @@ async def async_step_user(
         client = get_async_client(self.hass)
         try:
             res = await client.get(user_input[CONF_URL], follow_redirects=True)
+            if res.status_code == HTTPStatus.FORBIDDEN:
+                errors["base"] = "forbidden"
```

Comment:
Or should we use this?
```suggestion
                errors["base"] = "invalid_auth"
```

Reply:
IMHO there is still a case for combining certain messages, if the underlying cause of this error is always that the user got the username/password wrong.

Do you have any more info about what condition causes this error?